server/plugin/api.go

```diff
@@ -21,7 +21,7 @@ import (
 
 const (
 	apiErrorIDNotConnected = "not_connected"
-	// the OAuth token expiry in seconds
+	//TokenTTL  is the OAuth token expiry duration in seconds
```

Comment:
Please, put an space between the `//` and the first word.
Also, there are two spaces between `TokenTTL` and `is`. Remove one.